src/libraries/System.Text.Json/Common/JsonObjectCreationHandling.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+
+    /// <summary>
+    /// Indicates how .NET properties or fields should be populated during deserialization.
+    /// </summary>
+#if BUILDING_SOURCE_GENERATOR
+    internal
+#else
+    public
+#endif
+    enum JsonObjectCreationHandling
+    {
+        /// <summary>
+        /// Member is replaced during deserialization.
+        /// </summary>
+        Replace = 0,
+
+        /// <summary>
+        /// Member is populated during deserialization.
```

Comment:
```suggestion
        /// Attempt to populate any instances already found on a deserialized field or property.
```

It might make sense to add  a <remarks> section providing a brief overview of what members do support this annotation.

Reply:
will be included in the next iteration